llvm/include/llvm/IR/IntrinsicsSPIRV.td

```diff
@@ -75,6 +75,11 @@ let TargetPrefix = "spv" in {
     [IntrNoMem] >;
   def int_spv_length : DefaultAttrsIntrinsic<[LLVMVectorElementType<0>], [llvm_anyfloat_ty], [IntrNoMem]>;
   def int_spv_normalize : DefaultAttrsIntrinsic<[LLVMMatchType<0>], [llvm_anyfloat_ty], [IntrNoMem]>;
+  def int_spv_refract
+      : DefaultAttrsIntrinsic<[LLVMMatchType<0>],
+                              [llvm_anyfloat_ty, LLVMMatchType<0>,
+                               llvm_anyfloat_ty],
```

Comment:
ETA is a scalar. this is not correct. what you want to do insteas is `LLVMVectorElementType<0>`